app/Http/Controllers/AccountController.php

```diff
@@ -138,8 +138,12 @@ public function updateEmail()
             ->fill(Request::input('user_email'));
 
         if ($userEmail->save() === true) {
+            $addresses = [$user->user_email];
             if (present($previousEmail)) {
-                Mail::to($previousEmail)->send(new UserEmailUpdated($user));
+                $addresses[] = $previousEmail;
+            }
+            foreach ($addresses as $address) {
+                Mail::to($address)->send(new UserEmailUpdated($user));
```

Comment:
Apparently you can send to multiple addresses by passing an object/collection to `Mail::to` (see https://laravel.com/docs/5.4/mail#sending-mail)